device/dhcp/servers/v4/dhcpv4serverLeases.yaml

```diff
@@ -0,0 +1,76 @@
+components:
+  schemas:
+    DhcpServerV4.Leases:
+      description: >-
+        Configuration for DHCPv4 Leases.
+      type: object
+      x-status:
+        status: under_review
+      properties:
+        lease_time: 
+          description: >-
+            The Life Time length in seconds that is assigned to a lease if the requesting DHCP client does not specify a specific expiration time.
+          type: integer
+          format: uint32
+          minimum: 300
```

Comment:
This looks very large for a test tool ... Might we not want to do tests where we keep retrying every 5/10 secs.

Reply:
True, changing to lower value. But min value in IxN side to be changed too.